docs-v2/content/development/setting-up-a-dev-environment.md

```diff
@@ -0,0 +1,106 @@
+# Setting up a dev environment (React)
+
+So you want to set up a dev environment? AWESOME!!! This doc is here to help you get set up as quickly and painlessly as possible so you can start contributing to storybook.
+
+## Prerequisites
+
+Please have the **_latest_** stable versions of the following on your machine
+
+-   node
+-   npm
+-   yarn
+-   create-react-app
+
+Please have at least version `3.3.0-alpha.0` of `@storybook/cli`
+
+## Setting it up
+
+### 1.  Setup storybook locally
+
+If you run into trouble here, make sure your node, npm, and **_yarn_** are on the latest versions.
+
+1.  `cd ~` (optional)
+2.  `git clone https://github.com/storybooks/storybook.git` _bonus_: use your own fork for this step
+3.  `cd storybook`
+4.  `yarn`
+5.  `yarn bootstrap --core`
+6.  `yarn test --core`
+
+#### Bootstrapping everything
+
+This method is slow, and if you are reading this tutorial you probably don't want to do it
+
+1.  `yarn bootstrap`
+2.  At the prompt: `a`, `enter`, `y`, `enter`
+3.  Have a beer 🍺
+4.  `yarn test`
+
+### 2. Create a project (skip this if you already have a react project)
+
```

Comment:
Most of the time this won't be needed. Using projects under `examples` directory is the easiest way of reproducing issues. In case of React it's `examples/cra-kitchen-sink`